src/protocolgame.cpp

```diff
@@ -2390,6 +2404,9 @@ void ProtocolGame::sendAddCreature(const Creature* creature, const Position& pos
 	msg.addByte(0x00); // can change pvp framing option
 	msg.addByte(0x00); // expert mode button enabled
 
+	msg.addString("http://static.tibia.com/images/store/");
+	msg.addByte(g_config.getNumber(ConfigManager::MAX_PACKETS_PER_SECOND));
```

Comment:
``MAX_PACKETS_PER_SECOND``

So.. 25?

How is this value related to the 10.90 update? Did something change with sendAddCreature and MAX_PACKETS_PER_SECOND?